### ISSUE TITLE
Can't find tox config file when running make test-unit

### ISSUE THREAD

**Issue by heisencoder** [fishtown-analytics/dbt#2739]
### Describe the bug

I'm trying to get my local developer setup by following the commands in https://github.com/fishtown-analytics/dbt/blob/dev/marian-anderson/CONTRIBUTING.md, but am running into issues with tox while running the unit tests.  

```
$ make test-unit
Unit test run starting...
Starting dbt_database_1 ... done
ERROR: tox config file (either pyproject.toml, tox.ini, setup.cfg) not found
Command exited with non-zero status 1
0.54user 0.11system 0:02.48elapsed 26%CPU (0avgtext+0avgdata 70524maxresident)k
0inputs+0outputs (0major+28516minor)pagefaults 0swaps
make: *** [Makefile:14: test-unit] Error 1
```

I can see tox.ini in my Git repo clone, so some other layer is expecting tox.ini elsewhere.  I can also get it fail when running docker directly:

```
$ docker-compose run test tox -e integration-postgres-py36 -- -x
Starting dbt_database_1 ... done
ERROR: tox config file (either pyproject.toml, tox.ini, setup.cfg) not found
```

### Steps To Reproduce
* Follow instructions at https://github.com/fishtown-analytics/dbt/blob/dev/marian-anderson/CONTRIBUTING.md
* run `make test-unit`

### Expected behavior
This should run the unit tests.

### System information
**Which database are you using dbt with?**
- [x] postgres

**The output of `dbt --version`:**
```
$ dbt --version
installed version: 0.18.0-rc1
   latest version: 0.17.2

Your version of dbt is ahead of the latest release!

Plugins:
  - postgres: 0.18.0rc1
  - redshift: 0.18.0rc1
  - snowflake: 0.18.0rc1
  - bigquery: 0.18.0rc1
```

**The operating system you're using:**
Debian 10 Linux variant

**The output of `python --version`:**
Python 3.8.5

### Additional context
I've installed docker and docker-compose, along with the other recommended tools.  This is reproducible in two of my Linux systems.

**Comment by heisencoder** [comment]
(See discussion in Slack on this topic at https://app.slack.com/client/T0VLPD22H/C50NEBJGG/thread/C50NEBJGG-1599003795.014400)

**Comment by jtcohen6** [comment]
Thanks for linking [that thread](https://getdbt.slack.com/archives/C50NEBJGG/p1599003795014400). This is _so_ outside my wheelhouse, but I'm supportive of any changes we can make here that, as @beckjake said, will not require env changes for the vast majority of users.

**Comment by heisencoder** [comment]
I've tested this fix for Linux, but don't know for sure if it works on Mac or Windows (although I expect it will).  The GitHub Actions should test against Windows, but I might need someone with a Mac to manually test the change locally on their system.

**Comment by jtcohen6** [comment]
Can do!

**Comment by jtcohen6** [comment]
resolved by #2768
